src/bindgen/ir/generic_path.rs

```diff
@@ -123,11 +235,15 @@ impl GenericPath {
         &self.export_name
     }
 
+    pub fn is_single_identifier(&self) -> bool {
+        self.generics.is_empty()
+    }
+
     pub fn rename_for_config(&mut self, config: &Config, generic_params: &GenericParams) {
         for generic in &mut self.generics {
             generic.rename_for_config(config, generic_params);
         }
-        if !generic_params.contains(&self.path) {
+        if !generic_params.iter().any(|param| param.name == self.path) {
```

Comment:
Might be worth implementing a proper `contains` in `GenericParams` perhaps, if this check needs to be done in many places... For now it's just two so it's probably fine.